src/effects/VST3/VST3Effect.cpp

```diff
@@ -329,64 +329,36 @@ bool VST3Effect::SupportsAutomation() const
 }
 
 bool VST3Effect::SaveSettings(
-   const EffectSettings &, CommandParameters & parms) const
+   const EffectSettings& settings, CommandParameters& parms) const
```

Comment:
It is documented that `SaveSettings` may assume `parms` is empty.  This is unlike `LoadSettings` which cannot assume `settings` begins empty.

Reply:
I assume that by this you mean that in `LoadSettings `I should add `else `branches to assign nullopt in case something could not be read, as you comment later.